### ISSUE TITLE
JG ADA/Accessibility

### ISSUE THREAD

**Issue by jennah619** [jennah619/QADocumentationPractice#39]
Device : DELL Laptop
Processor Intel(R) Core(TM) i5-5200U CPU @ 2.20GHz 2.20 GHz
System type 64-bit operating system, x64-based processor
Windows 10

Web Browser-Chrome-Version 93.0.4577.63 (Official Build) (64-bit)

Device : Apple IPad 6th Gen - IOS 14.8
Device: Apple Iphone SE- IOS 10

Website: https://jungroup.com

Description/Summary:
No ADA or Accessibility page/link on website

Expected Behavior:
ADA is not mandatory for private business, but is considered inclusive and becoming the "norm" for business in general.
A windows screen reader should be able to read entire website.

Actual Behavior:
Ran WAVE tool and found 109 Aria errors.

Suggest: Correcting ARIA errors for greater accessibility for all users. 

<img width="960" alt="JGWAVE" src="https://user-images.githubusercontent.com/79431576/210004301-8d45d240-569b-47f5-9703-a3dbe87e9bbe.png">

Suggest: Resolve Alt Text errors to make website more user friendly.
